packages/schema/src/config/build.ts

```diff
@@ -54,6 +54,24 @@ export default defineUntypedSchema({
     }
   },
 
+  /**
+   * Node.js polyfills for the client build (Browser).
+   * @see https://github.com/unjs/unenv
+   *
+   * **Note:** For making globals such as `Buffer` and `process` to work in Browser, you need to manually inject them.
+   *
+   * **Example:** plugins/node.client.ts:
+   *
+   * ```ts
+   * import { Buffer } from 'node:buffer'
+   * import process from 'node:process'
+   *
+   * globalThis.Buffer = globalThis.Buffer || Buffer
+   * globalThis.process = globalThis.process || process
+   * ```
+   */
+  unenv: {},
```

Comment:
should we type this?

Reply:
Ah somehow removed in last comit will revert